### ISSUE TITLE
Event not triggered when state changes while using accessibilityLiveRegion

### ISSUE THREAD

**Issue by vineethkuttan** [microsoft/react-native-windows#14608]
### Problem Description

So as per the code for the Switch Component 
[react-native-windows/packages/@react-native-windows/tester/src/js/examples-win/Switch/SwitchExample.windows.js at main · microsoft/react-native-windows](https://github.com/microsoft/react-native-windows/blob/main/packages/%40react-native-windows/tester/src/js/examples-win/Switch/SwitchExample.windows.js#L291)

there are four toggle button .
 
The first button (1,1) doesn't have liveregion prop
Second one  (1,2)  has liveregion prop set to none
Third one (2,1)  has liveregion prop set to polite
Fourth one (2,2)  has liveregion prop set to assertive

But when using narrator , all the buttons are behaving in the same way

https://github.com/user-attachments/assets/0f705da0-59fd-499e-8b5c-e83fc92f3de5

 

### Steps To Reproduce

Run RNTester App ,Navigate to the switch component , and open the narrator app and try

### Expected Results

Narrator should narrate 3rd and 4th button when its state changes multiple times.

### CLI version

15.0.0-alpha.2

### Environment

```markdown
System:
  OS: Windows 11 10.0.26100
  CPU: "(16) x64 AMD EPYC 7763 64-Core Processor                "
  Memory: 40.04 GB / 63.95 GB
Binaries:
  Node:
    version: 22.14.0
    path: C:\Program Files\nodejs\node.EXE
  Yarn:
    version: 1.22.22
    path: C:\ProgramData\global-npm\yarn.CMD
  npm:
    version: 10.9.2
    path: C:\Program Files\nodejs\npm.CMD
  Watchman: Not Found
SDKs:
  Android SDK: Not Found
  Windows SDK:
    AllowDevelopmentWithoutDevLicense: Enabled
    Versions:
      - 10.0.19041.0
      - 10.0.22621.0
IDEs:
  Android Studio: Not Found
  Visual Studio:
    - 17.13.35931.197 (Visual Studio Enterprise 2022)
Languages:
  Java: Not Found
  Ruby: Not Found
npmPackages:
  "@react-native-community/cli": Not Found
  react: Not Found
  react-native: Not Found
  react-native-windows: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: Not found
  newArchEnabled: Not found
iOS:
  hermesEnabled: Not found
  newArchEnabled: Not found
```

### Target Platform Version

None

### Visual Studio Version

None

### Build Configuration

None

**Comment by chrisglein** [comment]
Here's [accessibilityLiveRegion](https://reactnative.dev/docs/accessibility#accessibilityliveregion-android).

Question is whether this is bugged behavior, not yet implemented, or behaving to spec with how focus works here.
@chiaramooney @Yajur-Grover Can you help identify where we are with live region?

**Comment by chiaramooney** [comment]
We've implemented sending accessibilityLiveRegion data to UIA, but because of the past breaks in Narrator we are only now seeing the additional event handling that's needed for Narrator to correctly announce event changes. This is a behavior gap that we should fix.

**Comment by vineethkuttan** [comment]
We Should not use LiveSetting prop on Switch component in the first place. Cause screen reader won't look for it for switch comp, when IToggleProvider is implemented.

Also here we missed to notify client when toggle state changes, the above PR resolves it.
